### PR TITLE
Add initial program for June

### PR DESCRIPTION
**Pull request by TueHenriksen**

### CODE REVIEW

File: meetups/2024/2024-06-20-talk-night/program.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+[INFORMATION ABOUT THE SPEAKER]
+
+
+## Architecturing services with dependency injection
+
+By [Ivan Luchev][il]
+
+Ivan will talk about how one can architecture an app following the dependency inversion principle. We will do this by building a simple app made out of modules that interact with each other by relying on their interfaces. To achieve this, we will talk about dependency injection — how to use it, common patterns, and pitfalls.
+
+Ivan Luchev is...
```
Comment (luchev) [inline meetups/2024/2024-06-20-talk-night/program.md:34]: Ivan has been a Software Engineer at Trifork for the last 2 years, working on a Data Platform. He's coming from a background of large scale data processing with Go. His experience with Rust spans 3 years, mainly working on side projects and research.

File: meetups/2024/2024-06-20-talk-night/program.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+[OR   We are still working on the program, but it will be something like the following:]
+
+- 18:05 - 18:15: Welcome (10 min)
+- 18:20 - 18:50: Free slot for a talk(s)
+- 18:50 - 19:00: A word from our sponsor (5-10 min)
+- 19:00 - 19:45: Light dinner sponsored by Trifork
+- 19:45 - 20:35: Architecturing services with dependency injection by Ivan Luchev (40 min + 10 min Q&A)
+- 20:40 - 20:50: 
+- 20:50 - 21:00: Closing remarks (10 min)
+- 21:00: Let's continue the discussion at the nearby bar.
```
Comment (TueHenriksen) [inline meetups/2024/2024-06-20-talk-night/program.md:17]:
```suggestion
- 21:00: Let's continue the discussion at a nearby bar.
```

File: meetups/2024/2024-06-20-talk-night/program.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+Rust Aarhus invites you to this free meetup for all interested in the programming language [Rust].
+
+The program for the evening is:
+[OR   We are still working on the program, but it will be something like the following:]
+
+- 18:05 - 18:15: Welcome (10 min)
+- 18:20 - 18:50: Free slot for a talk(s)
+- 18:50 - 19:00: A word from our sponsor (5-10 min)
+- 19:00 - 19:45: Light dinner sponsored by Trifork
```
Comment (TueHenriksen) [inline meetups/2024/2024-06-20-talk-night/program.md:13]:
```suggestion
- 19:00 - 19:45: Light dinner sponsored by [Trifork]
```

File: meetups/2024/2024-06-20-talk-night/program.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+## Location
+This meetup will be at Trifork, Sankt Knuds Torv 9, 2. floor.
+
+## Food
+We will try to have a combination of vegan and non-vegan food. If you have special needs regarding food (e.g. allergies), please forward your request to tue@rustaarhus.dk.
+
+
+[rust]: https://www.rust-lang.org/
+[il]: https://github.com/luchev
```
Comment (TueHenriksen) [inline meetups/2024/2024-06-20-talk-night/program.md:45]:
```suggestion
[il]: https://github.com/luchev
[trifork]: https://trifork.com/
```

File: meetups/2024/2024-06-20-talk-night/program.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+- 21:00: Let's continue the discussion at a nearby bar.
+
+## [TITLE OF FIRST TALK]
+
+By [SPEAKER]
+
+[ADD DESCRIPTION OF THE TALK HERE]
+
+[INFORMATION ABOUT THE SPEAKER]
+
```
Comment (jbethune) [inline meetups/2024/2024-06-20-talk-night/program.md:26]:
```suggestion
## Creating Rust-Powered Text Adventures

By Jörn

You read the description of a meetup talk. Apparently the talk is about open world text adventures. As your hands fly your keyboard to search for the right keywords, you notice that there is friendly crab sitting in the corner of the screen. You rest your hands in mid-air as the crab starts to speak: "Have you thought about writing your own text adventures using a cool new Rust-based stack?" What do you say?

* Sounds awesome, tell me more! -> Go to section 42
* I want to play some games! -> Go to section 100
* I'm just here for the free food and the hygge -> Go to section 77


Jörn is a bioinformatician who is curious about many things living and digital.

```

**Pull request merged**